docs/query-guide/functions.rst

```diff
@@ -102,6 +102,14 @@ math, string manipulation or more sophisticated expressions to be expressed.
 
     Returns true if ``b`` is a substring of ``a``
 
+.. function:: strip(s[, leading, trailing])
```

Comment:
Thanks @reidgilman!
I think we talked out-of-band, but I think out of preference, it's cleaner to have `lstrip()` `strip()` and `rstrip()` as separate functions instead of multiple flags.

Reply:
Resolved in 2adb5a0